freq/lizard_connector.py

```diff
@@ -343,34 +399,15 @@ def all_to_csv(self, start='0001-01-01T00:00:00Z', end=None,
         if isinstance(end, int):
             end += 10000
         org_query = self.organisation_query(organisation)
-        self.get(
-                start=start,
-                end=end,
-                **org_query
-            )
-        csv = (
-            [r['name'], r['uuid'], jsdatetime.js_to_datestring(e['timestamp']), e['max']] for r
-            in self.results for e in r['events']
-        )
-        loc = Locations(use_header=self.use_header)
-        extra_queries_ts = copy.deepcopy(self.extra_queries).items()
-        extra_queries = {
-            key if not key.startswith("location__") else key[10:]: value
-            for key, value in extra_queries_ts
-        }
-        org_query = self.organisation_query(organisation, '')
-        extra_queries.update(**org_query)
-        loc.get(**extra_queries)
-        coords = loc.coord_uuid_name()
-        headers = (
-            [
-                r['uuid'], r['name'], coords[r['location']['uuid']]['name'],
-                coords[r['location']['uuid']]['coordinates'][0],
-                coords[r['location']['uuid']]['coordinates'][1]
-            ]
-            for r in self.results
-        )
-        return headers, csv
+        poll_url = self.get(
+            start=start,
+            end=end,
+            async="true",
+            format="json",
+            **org_query
+        )[0]['url']
```

Comment:
een raise api error zou netter zijn ja.Maar als hij er hier uitklapt is er iets goed mis.